types/mirrorx/tslint.json

```diff
@@ -0,0 +1,81 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Ooops, I missed this in my last review. Please don't turn off an lint rules on new packages, they're there to do good!

Reply:
I have turned these rules on . Please check. Thanks. @minestarks

Reply:
some are still off ,  because people are different.

Reply:
The linter rules enforce the coding standards for this repo to make it easier for everybody to read and maintain these typings. Could you turn them back on before I can merge this, please? Thank you!

Reply:
OKey.

Reply:
Hi,@minestarks  I have some questions. How can I contact you? E-mail or skype? 
I install the extension "tslint" in vscode. However, it does not work. Is there any other way to auto-fix the errors ?
And the tests.ts require the module 'mirrox' in npm. 
How to add the mirrorx to the whitelist in travis?

Reply:
@minestarks  this has taken lots of my time. I just want to finish it. Please  help to solve my puzzles above!

Reply:
Sorry about the confusion :/

This is the best place to contact me, and others who can also help.

I don't know what you mean by the npm dependency. I'm adding a comment to your line with the `import` statement -- I think that should help?

You can look at the Travis CI log here on GitHub to see the errors. 

Or to run the linter locally, from your DefinitelyTyped repo, run:

```
npm install
npm run lint mirrorx
```

In vscode, do Alt-Shift-F to format the whole document, that will fix a lot of the whitespace errors.

Most of the other errors are fairly obvious, you can fix them line by line. I wish there was a way to auto-fix them, but I don't know of any!